app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -48,13 +48,20 @@ const ExperimentDetails: React.FC = () => {
     // No-op
   };
 
+  let bugzilla_url;
+  if (data.bugzilla_url) {
```

Comment:
Sorry, I'm not 100% on serializers and the backend of this... but what are the circumstances around there _not_ being a `bugzilla_url`?
Is it just a timing issue where I create a new rapido experiment and then hit "Next" while in draft and the bug hasn't been created in bugzilla yet due to timing issues, but would always be there on a subsequent page refresh?

Reply:
^ this would be a case where there isn't a bugzilla_url. The clunkiness of needing to refresh will be addressed in diff issue/PR

another reason why there wouldn't be a bugzilla url would be if the bugzilla api for some reason fails?

Reply:
Ah, thanks. I think I was confusing it with a test above where were concatenating a `bug_id` onto `bug_host` and was thinking it'd concatenate a null/undefined `bug_id` and we'd have a URL like `https://bugzilla.whatever/show_bug.cgi?id=` (which would never be falsy) but now I see you're storing the entire URL and not just the `bugzilla_id` (or maybe you're storing both).